dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -160,12 +169,19 @@ def __init__(
       self._structure = structure.convert_legacy_structure(
         self._dtypes, self._shapes, output_classes)
 
-      if _get_tf_minor_version() == '14':
+      if _get_tf_version() >= StrictVersion('1.14'):
         super(_DALIDatasetV2, self).__init__(self._as_variant_tensor())
-      elif _get_tf_minor_version() == '13':
+      elif _get_tf_version() >= StrictVersion('1.13') and _get_tf_version() < StrictVersion('1.14'):
```

Comment:
I think this condition is always true here.
Because L172 `_get_tf_version() < StrictVersion('1.14')` and due to L133 `_get_tf_version() >= StrictVersion('1.13')`

Reply:
Done